src/components/ebay-notice/examples/11-window-notice/template.marko

```diff
@@ -0,0 +1,10 @@
+<ebay-notice
+    type="window"
+    a11y-heading-text="information">
```

Comment:
I think `a11y-heading-text` becomes irrelevant with window notice, because it just uses the entire contents of the h2 to label the region. We'll clean up these differences when we refactor, but for now just make a note of it in docs and remove from example.